Flipkart-price-alert/README.md

```diff
@@ -0,0 +1,22 @@
+# Flipkart Price Alert
+Checks for the price of a desired product periodically.
+If the price drops below the desired amount, notifies the user via email.
+# Setup Instructions
```

Comment:
All heading levels after the "# Flipkart Price alert" heading need to be on the same level. Either make them (##) or (#)